### PR TITLE
Displays raiden account balance in general screen when using subkey

### LINKED ISSUE #1300: I want to see my Raiden Account and ETH balance on the "General Screen"
## User Story
As Chris I want to see my Raiden Account ETH balance on the "General Screen" so that I see my "hidden" account and balance.

Split from #1108 

## Acceptance criteria
- Raiden Account ETH balance is shown when using the subkey
- ETH Account ETH balance is shown when using the main key

### PR DESCRIPTION
**Pull request by kelsos**
Closes #1300

### CODE REVIEW

File: raiden-dapp/CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,12 @@
 
 ### Fixed
 
+- [#1300] General screen shows Raiden account balance when using subkey.
```
Comment (nephix) [inline raiden-dapp/CHANGELOG.md:11]: Wait why is this still unreleased?

Comment (nephix) [reply raiden-dapp/CHANGELOG.md:11]: I think the change logs haven't been updated on v0.6.0 release

Comment (kelsos) [reply raiden-dapp/CHANGELOG.md:11]: Yes I need to rebase give me a moment

Comment (nephix) [reply raiden-dapp/CHANGELOG.md]: Ah ok thank you 🙏

**Pull request merged**